docs/topics/spider-middleware.rst

```diff
@@ -255,7 +255,8 @@ this::
 The ``handle_httpstatus_list`` key of :attr:`Request.meta
 <scrapy.http.Request.meta>` can also be used to specify which response codes to
 allow on a per-request basis. You can also set the meta key ``handle_httpstatus_all``
-to ``True`` if you want to allow any response code for a request.
+to ``True`` if you want to allow any response code for a request, and to ``False``
+to disable it.
```

Comment:
I think ‘disable it’ here is slightly ambiguous. Maybe ‘disable the effects of the ``handle_httpstatus_list`` key’?

Reply:
I agree, I am changing it